
From: Hans Reiser <reiser@namesys.com>

Reiser4 manipulates with super block's inode lists.  To do that it needs
inode_lock and __iget to be external symbols.

Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/fs/inode.c |    2 ++
 1 files changed, 2 insertions(+)

diff -puN fs/inode.c~reiser4-export-inode_lock fs/inode.c
--- 25/fs/inode.c~reiser4-export-inode_lock	2005-01-12 23:15:23.955652416 -0800
+++ 25-akpm/fs/inode.c	2005-01-12 23:15:42.704802112 -0800
@@ -81,6 +81,7 @@ static struct hlist_head *inode_hashtabl
  * the i_state of an inode while it is in use..
  */
 DEFINE_SPINLOCK(inode_lock);
+EXPORT_SYMBOL(inode_lock);
 
 /*
  * iprune_sem provides exclusion between the kswapd or try_to_free_pages
@@ -235,6 +236,7 @@ void __iget(struct inode * inode)
 		list_move(&inode->i_list, &inode_in_use);
 	inodes_stat.nr_unused--;
 }
+EXPORT_SYMBOL(__iget);
 
 /**
  * clear_inode - clear an inode
_
